applications/sofa/gui/qt/RealGUI.cpp

```diff
@@ -1686,26 +1680,13 @@ void RealGUI::initViewer(BaseViewer* _viewer)
 
 //------------------------------------
 
-void RealGUI::parseOptions(const std::vector<std::string>& options)
+void RealGUI::parseOptions()
 {
-    for (unsigned int i=0; i<options.size(); ++i)
-    {
-        if (options[i] == "enableInteraction")
-            m_enableInteraction = true;
-        if (options[i] == "noViewers")
-            mCreateViewersOpt = false;
-        if (options[i].substr(0,4).compare("msaa") == 0)
-        {
-            std::string::size_type pos = options[i].find('=') + 1;
-            if(pos < options[i].npos)
-            {
-                std::string strNb = options[i].substr(pos);
-                m_viewerMSAANbSampling = atoi(strNb.c_str());
-                if(m_viewerMSAANbSampling < 2 || m_viewerMSAANbSampling > 32)
-                    m_viewerMSAANbSampling = 1;
-            }
-        }
-    }
+    po::variables_map vm = mArgumentParser->getVariableMap();
+    if(vm.find("enableInteraction") != vm.end())
+        m_enableInteraction = vm["enableInteraction"].as<bool>();
+    if(vm.find("msaa") != vm.end())
```

Comment:
@damienmarchal

Reply:
You can retrieve the option by following steps in this function. But I forgot the warning message :/
Let me know if it does not works

Reply:
No, because enableInteraction is not an option in the argument list it is interactive. 
I fixed that in the PSL branch.

Reply:
Oops. My bad, sorry.

Reply:
Thanks for the fix